docs/openapi/components/schemas/common/ExternalIdentifier.yml

```diff
@@ -0,0 +1,49 @@
+$linkedData:
+  term: ExternalIdentifier
+  '@id': https://w3id.org/traceability#ExternalIdentifier
```

Comment:
yikes to defining our own RDF class for this... we are killing interop with schema.org for no real benefit.

Reply:
Good catch, thanks!